chat-plugins/economy.js

```diff
@@ -64,98 +64,98 @@ exports.commands = {
 		if (userid.length > 19) return this.sendReply("/wallet - [user] can't be longer than 19 characters.");
 
 		Economy.readMoney(userid, money => {
-			this.sendReplyBox(SG.nameColor(target, true) + " has " + money + ((money === 1) ? " " + currencyName + "." : " " + currenyPlural + "."));
+			this.sendReplyBox(SG.nameColor(target, true) + " has " + money + ((money === 1) ? " " + currencyName + "." : " " + currencyPlural + "."));
 			//if (this.broadcasting) room.update();
 		});
 	},
 
-	gc: 'givecurrency',
-	givecurrency: function (target, room, user) {
+	gs: 'givestardust',
+	givestardust: function (target, room, user) {
 		if (!this.can('forcewin')) return false;
-		if (!target) return this.sendReply("Usage: /givecurrency [user], [amount]");
+		if (!target) return this.sendReply("Usage: /givestardust [user], [amount]");
 		let splitTarget = target.split(',');
-		if (!splitTarget[2]) return this.sendReply("Usage: /givecurrency [user], [amount], [reason]");
+		if (!splitTarget[2]) return this.sendReply("Usage: /givestardust [user], [amount], [reason]");
 		for (let u in splitTarget) splitTarget[u] = splitTarget[u].trim();
 
 		let targetUser = splitTarget[0];
-		if (toId(targetUser).length < 1) return this.sendReply("/givecurrency - [user] may not be blank.");
-		if (toId(targetUser).length > 19) return this.sendReply("/givecurrency - [user] can't be longer than 19 characters");
+		if (toId(targetUser).length < 1) return this.sendReply("/givestardust - [user] may not be blank.");
+		if (toId(targetUser).length > 19) return this.sendReply("/givestardust - [user] can't be longer than 19 characters");
 
 		let amount = Math.round(Number(splitTarget[1]));
-		if (isNaN(amount)) return this.sendReply("/givecurrency - [amount] must be a number.");
-		if (amount > 1000) return this.sendReply("/givecurrency - You can't give more than 1000 currency at a time.");
-		if (amount < 1) return this.sendReply("/givecurrency - You can't give less than one buck.");
+		if (isNaN(amount)) return this.sendReply("/givestardust - [amount] must be a number.");
+		if (amount > 1000) return this.sendReply("/givestardust - You can't give more than 1000 Stardust at a time.");
```

Comment:
We should be using `global.currencyPlural` here instead of a hardcoded word.

On SG it would say: `"/givecurrency - You can't give more than 1000 Stardust at a time."`
While on Tsunami it could say: `"/givecurrency - You can't give more than 1000 Gyrabucks at a time."`

Also we should instead of just saying `/givecurrency` is the command, we could do:
```javascript
if (amount > 1000) return this.sendReply("/" + cmd + " - You can't give more than 1000 Stardust at a time.");
```
so it says exactly what command the user used.
Though we would need to edit the function arguments to have cmd.